### DISCUSSION TITLE
Rider Completion Problem

### DISCUSSION THREAD

**Discussion by progesor** [community/community#120099] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Copilot completion has not been working since yesterday.
I downgrade to Rider, re-install to copilot.I try everythink.
I have visual studio code and copilot working perfect. In rider chat working but completion not work.
Giving this error: The pending stream has been canceled (caused by: unable to verify the first certificate)
[idea.log](https://github.com/community/community/files/15026924/idea.log)

**Comment by sabar27** [comment]
The same problem using IntelliJ Idea plugin from noonday. 

Code completion stopped working with the same error. IntelliJ Idea fully reinstalled, completion worked for few minutes and stopped worked again...
Copilot chat is working, also code completion working in Visual Studio Code...

**Comment by Yamazaki93** [comment]
If this was working before, I think something changed in [1.5.2.5345](https://plugins.jetbrains.com/plugin/17718-github-copilot/versions/stable/517133) that broke it. We have security software that does TLS/SSL scanning and that's causing the problem, disabling TLS/SSL filtering function fixed it for us.

**Comment by progesor** [comment]
I tried changing the plugin version. Stable to Nightly. I tried all versions compatible with IDE. At first it works for a very short time, then it gives an error.

**Comment by Yamazaki93** [comment]
Yep, we tried previous versions as well, same symptom, sometimes it'll get an initial copilot response, then "unable to verify first certificate" again. Reported to Github, hopefully they'll be able to fix it. VSCode works just fine so it's definitely the plugin's issue.

**Comment by pasindujr** [comment]
If you are using an anti-virus program, try disabling the SSL/TLS filter. It worked for me on PhpStorm and I have ESET Internet Security installed on my computer.

**Reply by GabrielM-A** [reply]
Thanks - that's got it working for me too
